README.md

```diff
@@ -75,6 +75,73 @@ The PMEM-CSI driver supports running in different modes, which can be controlled
 
 * **_Unified_** mode is intended to run the driver in a single host, mostly for testing the driver in a non-clustered environment.
 
+### Volume Persistency
+
+Normally, volumes are provided by a storage backend that is independent
+of a particular node. When a node goes offline, the volume can be
+mounted elsewhere. With PMEM-CSI, volumes are *local* and thus can
+only be used on the node where they were created.
+
+This limitation needs to be considered when designing and deploying
+applications that are to use local storage:
+
+* Persistent Volumes  
+A volume gets created independently of the application, on some node
+where there is enough free space. Applications using such a volume are
+then forced to run on that node and cannot run when the node is
+down. Data is retained until the volume gets deleted.
+* Ephemeral Volumes  
+An application is assigned to a node and when it starts to run, a new
+volume is created for it on that node. When the application stops, the
+volume is deleted.
+* Cache Volumes  
+Volumes are created on a certain set of nodes, each with its own local
+data. Applications are started on those nodes and then get to use the
+volume on their node. Data persists across application restarts. This
+is useful when the data is only cached information that can be
+discarded and reconstructed at any time *and* the application can
+reuse existing local data when restarting.
+* Delayed Persistent Volumes  
+This is a variant of persistent volumes. The only difference is that
+creating the volume is delayed until the first application using the
+volume is started, and then the volume is created on the node chosen
+for that first application run. From then on, applications will always
+have to run on that node.
+
+Not all of these modes are supported in CSI, Kubernetes and PMEM-CSI. This table shows the current status and known limitations:
+
+Volume | Kubernetes | PMEM-CSI | Limitations
+--- | --- | --- | ---
+Persistent | supported | supported | topology aware scheduling
+Ephemeral | [in design](https://github.com/kubernetes/enhancements/blob/master/keps/sig-storage/20190122-csi-inline-volumes.md#proposal) | supported via a fake volume | topology aware scheduling, resource constraints
+Cache | supported | not implemented | topology aware scheduling
+Delayed Persistent | supported, [CSI support might be faulty](https://github.com/kubernetes-csi/external-provisioner/issues/221) | not implemented | topology aware scheduling, topology aware volume provisioning
```

Comment:
The issue on "external-provisioner" does not solve the "Delayed Persistent", most close to Ephemeral.  In my opinion the "Delayed Persistent" is the result of lack of  topology awareness in CSI that time. It was a work around solution which i don't think should carry longer, once we implement the complete toplogy awareness while creating volume, this work around can be dropped.